docs/manual/layout/templates/_index.en.md

```diff
@@ -6,16 +6,28 @@ aliases:
 weight: 40
 ---
 
-A template is used to output a module, content element, form or other component and contains HTML and PHP code. In the navigation area "Layout" under "Templates" the files can be created, stored in folders and edited. These adjustments are update proof.
+A template is used to output a module, content element, form or other component. In the navigation area "Layout" 
+under "Templates" the files can be created, stored in folders and edited. These adjustments are update proof.
```

Comment:
To keep PRs easier to review, don't include any unrelated changes like this. Only change the content and leave any existing formatting as is. If you want to change markdown formatting, create a separate PR please.

Reply:
Regarding the mentioned paragraph, I wanted to keep the description more general, just because of the new, additional Twig functionality.

Furthermore, I understood your hint and will take this into account in the future.

Reply:
That particular paragraph has no changes in your PR. The only change is that you added a line break after `"Layout"`, that's what I mean.

Reply:
Thank you.
Maybe i do have check my VSCode Setup regarding: 
"Add line breaks after 80 characters in paragraphs." (contribute.md)

Reply:
Oh does it do that then automatically for any file that you edit? Yeah, I think you need to change that ;). Personally I only added the ruler for `140` characters. 

`.vscode/settings.json`

```json
{
    "editor.rulers": [
        {
            "column": 140
        }
    ]
}
```

I also wanted to change the guideline to 140 characters in general, since we currently have a mix anyway.

Reply:
Will check.
So 140 will be the official

Reply:
Changed info in #946